addon/components/lottie.ts

```diff
@@ -76,13 +77,35 @@ export default class LottieComponent extends Component<LottieArgs> {
 
     const speed = Ember.testing ? 0 : this.args.speed || 1;
     this.animation.setSpeed(speed);
+
+    this.mediaQuery?.addEventListener(
+      'change',
+      this.handleReducedMotionPreferenceChange
+    );
   }
 
   willDestroy() {
     super.willDestroy();
 
+    this.mediaQuery?.removeEventListener(
+      'change',
+      this.handleReducedMotionPreferenceChange
+    );
+
     if (this.animation) {
       this.animation.destroy();
     }
   }
+
+  private handleReducedMotionPreferenceChange = () => {
```

Comment:
sand: this is maybe just my preference so please resolve as you like but you could avoid an anonymous function and bind to this doing the following

```js
@action private handleReducedMotionPreferenceChange() {
```

Reply:
I did the change, it looks more "ember style".
Now that I'm thinking about it, this is not a real "action" BTW; I believe the `@action` decorator is more for callbacks triggered by user actions, isn't it? But anyway it doesn't make a real difference, I like it more like you suggested 👍

Reply:
Yeah mostly used for binding function to component for calling from templates. Not sure what other peoples thought on using it like this 😅

Reply:
Seems a fair way to use it anyway 👍